ecommerce_api_client/tests/test_client.py

```diff
@@ -20,7 +21,7 @@ class EcommerceApiClientTests(TestCase):
 
     def test_valid_configuration(self):
         """ The constructor should return successfully if all arguments are valid. """
-        EcommerceApiClient(URL, SIGNING_KEY, USERNAME, EMAIL)
+        EcommerceApiClient(URL, SIGNING_KEY, USERNAME, FULL_NAME, EMAIL)
 
     @ddt.data(
         {'url': None, 'signing_key': SIGNING_KEY, 'username': USERNAME, 'email': EMAIL},
```

Comment:
Assuming you limit the requirements gating use of JWT authentication to `signing_key` and `username`, you'll have to update the test data here accordingly.